topdown/builtins/builtins.go

```diff
@@ -57,31 +66,28 @@ func (c NDBCache) Get(name string, k ast.Value) (ast.Value, bool) {
 
 // Convenience functions for serializing the data structure.
 func (c NDBCache) MarshalJSON() ([]byte, error) {
```

Comment:
When the NDBCache was first developed, I'd thought I needed to use `ast.Term` types to wrapper over AST values. I was wrong, and it turns out you *can* serialize raw `ast.Value` types (like `ast.Object`), but you have to be careful about how you do it, especially on the deserialization side.

The changes in JSON ser/des for the NDBCache result in much cleaner JSON blobs, which save a lot of space in decision logs, and are easier to read too! :smile: